recipes/youtube-sub-extractor

```diff
@@ -0,0 +1 @@
+(youtube-sub-extractor :fetcher github :repo "agzam/youtube-sub-extractor.el" :files ("*.el"))
```

Comment:
```suggestion
(youtube-sub-extractor :repo "agzam/youtube-sub-extractor.el" :fetcher github)
```

Reply:
(This recipe is equivalent and gives your repo room to grow without a followup PR.)

Reply:
The reason for :files filter is that I wanted to exclude tests. If you think I don't have to do that, I can remove it.

Reply:
Check out the [default value](https://github.com/melpa/melpa/blob/master/README.md#recipe-format) for the recipe file selector: it will exclude files that end in -test.el or -tests.el and has a lot of other conventions.